pkg/experiment/metastore/index/models.go

```diff
@@ -0,0 +1,72 @@
+package index
+
+import (
+	"flag"
+	"sync"
+	"time"
+
+	"github.com/go-kit/log"
+
+	metastorev1 "github.com/grafana/pyroscope/api/gen/proto/go/metastore/v1"
+)
+
+var DefaultConfig = Config{
+	PartitionDuration: time.Hour,
```

Comment:
I understand that this is for testing, but just to make sure we're on the same page: I believe partition duration should be substantially longer than the expected block duration (after compaction)